app/utils/calculate-burn-height.ts

```diff
@@ -0,0 +1,16 @@
+import BigNumber from 'bignumber.js';
+
+interface CalcBurnHeightBlockFromCyclesArgs {
+  cycles: number;
+  rewardCycleLength: number;
+  currentBurnBlockHeight: number;
+  blocksToNextCycle: number;
+}
+export function calculateUntilBurnHeightBlockFromCycles(args: CalcBurnHeightBlockFromCyclesArgs) {
```

Comment:
This calculation looks good to me -- I'd just recommend adding a unit test with the mainnet cycle length (2100) as well.